src/view/src/main/java/airqo/models/Insight.java

```diff
@@ -43,6 +45,20 @@
 	private Frequency frequency;
 	private String siteId;
 
+	@Transient
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = dateTimeFormat, timezone = "UTC")
+	@Getter(value = AccessLevel.PRIVATE)
+	@DateTimeFormat(pattern = dateTimeFormat)
+	@JsonIgnore
+	private Date startDateTime;
+
+	@Transient
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = dateTimeFormat, timezone = "UTC")
+	@Getter(value = AccessLevel.PRIVATE)
+	@DateTimeFormat(pattern = dateTimeFormat)
+	@JsonIgnore
+	private Date endDateTime;
```

Comment:
Added to fields, `startDateTime` and `endDateTime` to use while querying start and end time resp.